pkg/reconciler/githubsource/webhook_client.go

```diff
@@ -88,8 +88,8 @@ func (client gitHubWebhookClient) Delete(ctx context.Context, options *webhookOp
 	if alternateGitHubAPIURL != "" {
 		baseURL, err := url.Parse(alternateGitHubAPIURL)
 		if err != nil {
-			logger.Infof("Failed to delete webhook - Error occured parsing githubAPIURL %s, error was %v", alternateGitHubAPIURL, err)
-			return fmt.Errorf("Error occured parsing githubAPIURL: %v", err)
+			logger.Infof("failed to delete webhook - error occured parsing githubAPIURL %s, error was %v", alternateGitHubAPIURL, err)
```

Comment:
Log messages (as opposed to errors that are returned) should be full sentences with capitalization and punctuation.

Reply:
Modified in latest commit